MAESTRO/Snakemake/scATAC/Snakefile

```diff
@@ -562,50 +497,35 @@ if config["platform"] == "10x-genomics" or config["platform"] == "sci-ATAC-seq":
                 shell:
                     "python " + SCRIPT_PATH + "/scATAC_FragmentGenerate.py -B {input.bam} -O {params.outdir} --CBtag CB --count;"
 
-            rule scatac_rmdp:
+            rule scatac_fragmentsample:
```

Comment:
I think we need to keep the mark duplicate step, but rename scatac_rmdp to scatac_markdp
and the bam file to "Result/minimap2/%s.sortedByPos.markdp.CBadded.bam".
because in the later bulk qc https://github.com/liulab-dfci/MAESTRO/blob/master/MAESTRO/Snakemake/scATAC/Snakefile#L154 we used samtools flagstat to get the read stats, and the duplication rate need to be there.